docs/en/Architecture-DataFlow.md

```diff
@@ -67,14 +65,24 @@ with other components; it only responds to requests via RPC services.
 The leading master records all file system transactions to a distributed persistent storage
 to allow for recovery of master state information; the set of records is referred to as the journal.
 
-#### Standby Masters
+### Standby Masters
+
+Standby masters are launched on different servers to provide fault-tolerance when running Alluxio
+master in HA (high available) mode. Standby masters read journals written by the leading master to
+keep their own copies of the master state up-to-date. They also write journal checkpoints for
+faster recovery in the future. However, they do not process any requests from other
+Alluxio components. After leading master fail-over, the standby masters will re-elect the new
+leading master after.
 
-Standby masters read journals written by the leading master to keep their own
-copies of the master state up-to-date. They also write journal checkpoints for
-faster recovery in the future. They do not process any requests from other
-Alluxio components.
+### Secondary Masters
 
-### Worker
+When running Alluxio master locally without HA mode, one can start a secondary master on the same
```

Comment:
Grammar: "**the** Alluxio master locally"

Reply:
fixed